choicemodels/tools/mergedchoicetable.py

```diff
@@ -348,13 +365,51 @@ def _build_table(self):
             samp_size = self.sample_size - 1
         
         obs_ids = np.repeat(self.observations.index.values, samp_size)
-        
+
+        # STRATIFIED SAMPLING OF ALTS: for now we are only supporting stratified sampling
+        # with replacement and no sampling weights
```

Comment:
Why is this restriction needed? How hard is it to support the other configs?

Reply:
Not hard, just wasn't needed for my use case so I didn't spend the time to write the extra code.